lib/ui/semantics.dart

```diff
@@ -547,8 +559,18 @@ class SemanticsUpdateBuilder extends NativeFieldWrapperClass2 {
     Float64List transform,
     Int32List childrenInTraversalOrder,
     Int32List childrenInHitTestOrder,
+    Int32List customAcccessibilityActions,
   ) native 'SemanticsUpdateBuilder_updateNode';
 
+  /// Update the custom accessibility action associated with the given `id`.
+  /// 
+  /// The name of the action exposed to the user is the `label`. The text 
+  /// direction of this label is the same as the global window.
+  void updateAction({int id, String label}) {
+    _updateAction(id, label);
```

Comment:
is "null" a valid value for both of these?

Reply:
No, same as updateNode.  Do you want asserts or ArgumentErrors here?

Reply:
asserts, thanks

Reply:
Done